src/radix_connect/p2p_links/p2p_links.rs

```diff
@@ -33,7 +33,6 @@ mod tests {
         assert_ne!(P2PLinks::sample(), P2PLinks::sample_other());
     }
 
-    #[test]
```

Comment:
intentional disabling of test? if so prefer `[ignore]`, maybe an accident? if it is broken, lets try to fix it.